src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -181,6 +182,29 @@ public function setValue(&$objectOrArray, $propertyPath, $value)
         }
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getValues(array $array, $propertyPath): array
+    {
+        $values = [];
+        foreach ($array as $item) {
+            try {
```

Comment:
`try/catch` blocks should be as specific as possible.

For eg., in that case, 
```
if (null === $value) {
    continue;
}

$values[] = $value;
```

Don't need to be inside the block.